### DISCUSSION TITLE
There is a way to hot reload quickly Blazor UI? Something like React hot reload? (blazor server-side)

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [dotnet/aspnetcore#29498] [Q&A] [answered]
Hello guys! 

There is a way to hot reload quickly Blazor UI? 
Something like React hot reload? (blazor server-side) 

Thanks so much!

**Comment by Pilchie** [comment, marked as answer]
We're working on it, see https://github.com/dotnet/aspnetcore/issues/5456.
